src/ImuSensor.cc

```diff
@@ -71,6 +71,15 @@ class ignition::sensors::ImuSensorPrivate
   /// \brief Flag for if time has been initialized
   public: bool timeInitialized = false;
 
+  /// \brief Store world heading offset
+  public: ignition::math::Quaterniond headingOffset;
```

Comment:
rename `headingOffset` to `worldRelativeOrientation` and update doc-string to "Orientation of world frame relative to a specified frame"

Reply:
Done